.github/workflows/rosetta-api.yml

```
@@ -97,6 +99,14 @@ jobs:
           [[ $code -eq 0 ]] && { wait $pid || code=$?; }
           cat /tmp/construction-validation.log
           exit $code
+
+      - name: Install Postman newman CLI
+        run: npm install -g newman
+
+      - name: Run Rosetta Postman API Tests
+        working-directory: ./hedera-mirror-rosetta/scripts/validation/postman
+        run: newman run rosetta-api-postman.json --env-var base_url=http://localhost:5700
```

nit: Can omit since it's the default
```suggestion
        run: newman run rosetta-api-postman.json
```

---

Omitted